### PR TITLE
DATA: Intern Model & Migrations

### LINKED ISSUE #26: DATA: `Intern` Model & Migrations
Go look for videos that do this :)

### PR DESCRIPTION
**Pull request by Manasa1191**
closes #26

### CODE REVIEW

**Comment by hassanhabib** [comment]
@Manasa1191 PR names _must_ match issue name including casing, colons and everything else.

File: InternTrack.Core.Api/Migrations/Configuration.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        public Configuration()
+        {
+            AutomaticMigrationsEnabled = false;
+        }
+
+        protected override void Seed(InternTrack.Core.Api.InternDbContext context)
+        {
+            //  This method will be called after migrating to the latest version.
+
+            //  You can use the DbSet<T>.AddOrUpdate() helper extension method
```
[the remark this answers is not in the document]
Comment (Manasa1191) [reply InternTrack.Core.Api/Migrations/Configuration.cs:19]: Looks like I installed incorrect Entity Framework package . I will send a new pull request with changes . 
I generated migrations using following video - Generating Model Migrations with Entity Framework.

File: InternTrack.Core.Api/appsettings.Development.json
Comment (Manasa1191) [inline InternTrack.Core.Api/appsettings.Development.json:1]: Only after changing 'appsettingsDevelopment.json' to  'appsettings.Development.json '  the build passed. Below is the migration error I got:  
image

File: InternTrack.Core.Api/Models/Interns/Intern.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
+        public string Status { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+        public DateTimeOffset JoinDate { get; set; }
+        public Guid CreatedBy { get; set; }
+        public Guid UpdatedBy { get; set; }
+        public String TimeZone { get; set; }
```
Comment (hassanhabib) [inline InternTrack.Core.Api/Models/Interns/Intern.cs:23]: Why do we need that?

Comment (hassanhabib) [reply InternTrack.Core.Api/Models/Interns/Intern.cs:23]: I talked to @Manasa1191 and she agreed and she will redo the migration
@ADILATIC make sure you start your PRs after @Manasa1191  is done.

**Pull request merged**